analysis/topEFT/datacard_maker.py

```diff
@@ -529,7 +523,7 @@ def processSyst(process, channel, systMap, d_hists, fout, cat):
                     syst_cat = 'parton'
                     jet = int(re.findall('\dj', channel)[0][:-1])
                     offset = -4 if '3l' not in channel else -2
-                    syst = 1+self.fparton[f'{cat};1'][process.split('_')[0]].array()[jet+offset]
+                    syst = 1+self.fparton['tllq'][process.split('_')[0]].array()[jet+offset]
```

Comment:
@bryates could you comment on what this part of the code is doing? Sorry, it's not quite clear to me

Reply:
It was a mistake when loading the missing parton uncertainty in the master branch. The root file _only_ has `tllq` in the `TTree`, but the master branch was also trying to read `tHq`.

Reply:
Ah, wait, sorry if I'm confused. The missing parton uncertainty is already implemented here? I thought that was a separate pR

Reply:
Yes, at least a version of it is here
https://github.com/TopEFT/topcoffea/blob/a4b95a897ec6c3abf5d83486a4d185c63f4ddffa/analysis/topEFT/datacard_maker.py#L528-L537

Reply:
Can you clarify if this is just functionality that is in place for when we need it? Or if we are actually already calculating and including a missing parton uncertainty in the cards/templates?

Reply:
This actually loads the missing parton uncertainty and places it in the datacards.

Reply:
Ah, ok. So, I think we actually do not want to do that in this PR, since I think (based on the MM conversation) we're not sure if we are ready to include this (or even need it in the first place). 

Also, I'm now confused about the check you quoted above. How can the only change with respect to master be the tttt scale stuff when this branch also adds missing parton uncertainties?

Reply:
This branch does not include the missing parton. The missing parton is currently in the master branch.

Reply:
Ok, sorry, I must have missed that. When did we merge the missing parton uncertainty into master? I knew we had functionality for it, but so far (as of the limits I made for the TOP EFT meeting last week) we have actually been including this yet.

Reply:
In my `small fixes` [PR](https://github.com/TopEFT/topcoffea/pull/261)

Reply:
Ok... so the only thing I see mentioned on that PR related to this is "Fixed a typo in missing_parton.py". Is that typo fix really what took us from not including the missing parton uncertainty to including it?

Reply:
I'm trying to understand now why it wasn't giving as an issue when it got to `tHq` on the master branch.

Reply:
Ok, that sounds like a good thing to try to understand. Anyway, I think for now, we actually do not want to be including the missing parton uncertainty (in general I don't think we want to add any new uncertainties before fully testing, discussing, and deciding that they're ready). So if this uncertainty is currently being applied, then I think in this PR I would actually ask that you disable it for now (since I think based on the MM discussion, we're not sure it's ready yet, and also it's not clear we'll even need it for sure in the first place). But if you have other thoughts on this, of course feel free to let me know.

Reply:
Ok, I'll just comment out that block for now.

Reply:
Ok, sounds good, thanks! Could you also double check (if you haven't already done this) that the only thing that changes in the datacard and templates (with respect to your previous tests) is that the missing extra parton uncertainty is not included?

Reply:
Ok, I just double checked and nothing else has changed. I guess the missing parton stuff was not loading anyway, but commenting it out is clearer.

Reply:
Ok, thanks Brent! That's good news. I'll be happy to merge this PR once the CI finishes.